app/javascript/App/lib/redux/apiSagas.js

```diff
@@ -57,16 +57,15 @@ function handleError({ error }) {
   console.log(error)
 }
 
-function* handleResponse(callback, response) {
-  if (!response) return null
-  if (response.message) {
-    console.log(response.message)
-  }
-  let payload
-  if (response.payload) {
-    payload = response.payload
-  } else {
-    payload = response
+function* handleResponse(callback, response, url) {
+  const { type, payload, message } = response
+  if (!response) {
```

Comment:
do all these conditions get logged in rollbar? is there another way to handle these cases besides logging to the console?

Reply:
No they don't, will make an Asana ticket for that